### DISCUSSION TITLE
Multiple cars in same account

### DISCUSSION THREAD

**Discussion by dpoler** [teslamate-org/teslamate#3238] [General]
Hi!
Two cars, same Tesla account. Both are "Seen" by Teslamate, however... They show up as "???" -
![image](https://github.com/adriankumpf/teslamate/assets/6889265/e4847616-d92a-4de0-a4eb-7eff546db87e)

In addition I cannot select which one to view in Grafana. The spot in which I think the car selector would be just shows as null...
![image](https://github.com/adriankumpf/teslamate/assets/6889265/575a6402-b456-4d46-a7e5-b39fe1e19521)

Appreciate any help! 
Thank you.

**Comment by cwanja** [comment]
See #3183

**Reply by dpoler** [reply]
Thanks, not sure why my search didn't turn that up. Appreciate it.

**Comment by sahumada** [comment]
fwiw, I did a `docker-compose pull && docker-composer up -d` and the issue got fixed

**Reply by cwanja** [reply]
Would be curious if that sticks without a TeslaMate update. My gut tells me no, as others have stated in the linked thread about.

**Reply by tobiasehlert** [reply]
@cwanja; I'm pretty sure you need to upgrade to a later version, since there is a change that you'd need https://github.com/adriankumpf/teslamate/commit/6af97bafc8cfee6bd22911a1014dae8dcbe1f419.
There is not a new release yet on TeslaMate, but if using `edge` you have have it in place already.
